perception/lidar_obstacle_detection/centerpoint_backbone/onnx_centerpoint_backbone/definition.yaml

```
@@ -0,0 +1,62 @@
+version: 1
+enable_testing: true
+network:
+  filename: ./model_files/pts_backbone_neck_head_centerpoint.onnx
+  framework: ONNX
+  provenance: https://github.com/open-mmlab/mmdetection3d
+  training: Kitti 3D Object Dectection Evaluation 2017 Dataset
+  model_license: Apache-2.0
+  data_license: Creative Commons BV-NC-SA 3.0
```

nit
```suggestion
  data_license: Creative Commons BY-NC-SA 3.0
```
same in the other file